galicaster/mediapackage/mediapackage.py

```diff
@@ -23,6 +23,8 @@
 import re
 import time
 import os
+import dateutil
```

Comment:
You don't need to import `dateutil` if you are only using the parser.

However, I would use a more specific name as "parser" (such as "dateparser" or similar), or simply import `dateutil` and then use `dateutil.parser.parse`, because it is more legible.